config/khronos.css

```diff
@@ -363,7 +363,7 @@ b.button:after { content: "]"; padding: 0 2px 0 3px; }
 #content #toc > :first-child { margin-top: 0; }
 #content #toc > :last-child { margin-bottom: 0; }
 
-#footer { max-width: 100%; background-color: none; padding: 1.25em; }
+#footer { max-width: 100%; background-color: transparent; padding: 1.25em; }
```

Comment:
`none` is invalid value for `background-color`. `transparent` is the closest semantically (and also the default value &mdash; so alternatively it could be removed), so I guessed that is what author meant here.

Reply:
It comes from asciidoctor template from old Foundation. Might be a browser hack (or not).
Though probably benign enough, so I decided revert it.